### ISSUE TITLE
fix Makefile to unbreak build with newer go

### ISSUE THREAD

**Issue by landryb** [facette/facette#306]
With go 1.7 on OpenBSD; linking fails:

```
/usr/local/go/pkg/tool/openbsd_amd64/link: -X flag requires argument of the form importpath.name=value
```

I dont know in which version the -X foo=bar syntax was added and what version of go you're planning to support, but tweaking Makefile#L133->136 to use it fixed the issue for me.

**Comment by vbatoufflet** [comment]
Hi @landryb,

We'll update the Makefile. If I recall, support for `-X foo=bar` was added in Go 1.5. So it will become the minimal version required to build Facette.

**Comment by woodsb02** [comment]
I am having this same error message when trying to build facette 0.3.0 on FreeBSD with go-1.7.1.

Could you please let me know how you tweaked the Makefile to fix this problem?

**Comment by woodsb02** [comment]
The following patch works for me:
--- Makefile.orig       2016-10-01 10:18:24.965128000 +0800
+++ Makefile    2016-10-01 10:18:48.102780000 +0800
@@ -116,10 +116,10 @@
        @$(call mesg_start,$(notdir $@),Building $(notdir $@)...)
        @install -d -m 0755 $(dir $@) && $(GO) build \
                        -ldflags " \
-                               -X main.version $(VERSION) \
-                               -X main.buildDate '$(BUILD_DATE)' \
-                               $(PKG_LIST:%=-X github.com/facette/facette/%.version $(VERSION)) \
-                               $(PKG_LIST:%=-X github.com/facette/facette/%.buildDate '$(BUILD_DATE)') \
+                               -X main.version=$(VERSION) \
+                               -X main.buildDate='$(BUILD_DATE)' \
+                               $(PKG_LIST:%=-X github.com/facette/facette/%.version=$(VERSION)) \
+                               $(PKG_LIST:%=-X github.com/facette/facette/%.buildDate='$(BUILD_DATE)') \
                        " \
                        -tags "$(TAGS)" \
                        -o $@ cmd/$(notdir $@)/*.go && \

**Comment by vbatoufflet** [comment]
Hi,

I just push a fixed  version of the Makefile in 7ae8cc2e337eb047c2939cd86949a0003bdd0d3f.

Go 1.5 will now be the minimum required version to build Facette.
